ansible-collection/examples/backup_location/delete.yaml

```diff
@@ -25,18 +25,9 @@
             operation: DELETE
             api_url: "{{ px_backup_api_url }}"
             token: "{{ px_backup_token }}"
-            org_id: "{{ org_id | default('default') }}"
+            org_id: "{{ item.org_id | default('default') }}"
```

Comment:
org_id is from inventory/all. It is common across objects. don't have to add for each object. Please revert this.

Reply:
So, we dont have to pass org_id from our inventory/group_vars/dlete.yaml as well then.
Will remove that as well. 
@t-nidhin